lms/djangoapps/onboarding/forms.py

```diff
@@ -210,12 +211,9 @@ class Meta:
         }
         widgets = {
             'year_of_birth': forms.TextInput,
-            'country': forms.TextInput(attrs={"autocomplete": "off"}),
             'not_listed_gender': forms.TextInput(attrs={'placeholder': ugettext_noop('Identify your gender here')}),
-            'city': forms.TextInput(attrs={"autocomplete": "off"}),
-            'language': forms.TextInput(attrs={"autocomplete": "off"}),
-            'country_of_employment': forms.TextInput(attrs={"autocomplete": "off"}),
-            'city_of_employment': forms.TextInput(attrs={"autocomplete": "off"}),
+            'country_of_employment': forms.HiddenInput(),
```

Comment:
Please remove optional parenthesis from function calls, like forms.HiddenInput() to keep the code symmetry.